discovery/kubernetes/client_metrics.go

```diff
@@ -241,8 +218,6 @@ func (f *clientGoWorkqueueMetricsProvider) Register(registerer prometheus.Regist
 	registerer.MustRegister(clientGoWorkqueueAddsMetricVec)
 	registerer.MustRegister(clientGoWorkqueueLatencyMetricVec)
 	registerer.MustRegister(clientGoWorkqueueWorkDurationMetricVec)
-	registerer.MustRegister(clientGoWorkqueueUnfinishedWorkSecondsMetricVec)
-	registerer.MustRegister(clientGoWorkqueueLongestRunningProcessorMetricVec)
```

Comment:
Why did these have to go? Do they produce bogus results with MultiNamespaceListerWatcher?

Reply:
Nevermind I see they have been removed in the version of `client-go` you introduced